components/inlinealert/stories/template.js

```diff
@@ -0,0 +1,80 @@
+import { html } from 'lit-html';
+import { classMap } from 'lit-html/directives/class-map.js';
+// import { ifDefined } from 'lit-html/directives/if-definedjs';
+
+import { Template as Icon } from "@spectrum-css/icon/stories/template.js"
+import { Template as Button } from "@spectrum-css/button/stories/template.js"
+
+import "../index.css";
+
+export const Template = ({
+  rootClass = "spectrum-InlineAlert",
+  customClasses = [],
+  headerText,
+  text,
+  variant="neutral",
+  ...globals
+}) => {
+  const { express } = globals;
+
+  try {
+    if (!express) import(/* webpackPrefetch: true */ "../themes/spectrum.css");
+    else import(/* webpackPrefetch: true */ "../themes/express.css");
+  } catch (e) {
+    console.warn(e);
+  }
+
+  let iconName;
+  switch (variant) {
+    case 'info':
+      iconName = 'Info'
+      break;
+    case 'positive':
+      iconName = 'CheckmarkCircle';
+      break;
+    case 'notice':
+    case 'negative':
+    case 'closable':
+      iconName = 'Alert';
+      break;
+    default:
+      iconName = undefined;
+  }
+
+  const iconMarkup = typeof iconName !== 'undefined' ?
+    html`
+      ${Icon({
+        ...globals,
+        iconName
```

Comment:
For the icon to get the color, I think you'll need to pass:
```suggestion
        iconName,
        customClasses: [`${rootClass}-icon`],
```